src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,421 @@
+     * @param response the string of the user input
+     */
+    private static void instructionManager(String[] command, String response) {
+        try {
+            if (command[0].equalsIgnoreCase("add")) {
+                taskManager(command, response);
+            } else {
+                remove(command);
+            }
+        } catch (RemoveException e) {
+            System.out.println(e.getMessage());
+        } finally {
+            lineBreak();
+        }
+    }
+
+    /**
+     * Deciphers the specific task that the user wants to add to the list
+     *
+     * @param command  the set of instructions the user input in a String array
+     * @param response the string of the user input
+     */
+    private static void taskManager(String[] command, String response) {
+        String TaskType = command[1];
+        try {
+            if ((command.length == 2) || (command[2].isEmpty())) {
+                throw new TaskException();
+            } else if (command[1].equalsIgnoreCase("event")) {
+                Event(response);
+            } else if (command[1].equalsIgnoreCase("todo")) {
+                toDo(response);
+            } else if (command[1].equalsIgnoreCase("deadline")) {
+                deadLine(response);
+            }
+        } catch (TaskException e) {
+            System.out.println("please specify " + TaskType + " to add!");
+        } catch (TimeException e) {
+            System.out.println(e.getMessage());
+        }
+    }
+
+    /**
+     * Deciphers whether the user wants to do or undo a specific task
+     *
+     * @param command  the set of instructions the user input in a String array
+     * @param response the string of the user input
+     */
+    private static void doUndoManager(String[] command, String response) {
+        try {
+            if (command[0].equalsIgnoreCase("done")) {
+                done(response);
+            } else if (command[0].equalsIgnoreCase("undo")) {
+                undo(response);
+            }
+        } catch (DoneUndoException e) {
+            System.out.println(e.getMessage());
+        } finally {
+            lineBreak();
+        }
+    }
+
+    /**
+     * removes a task from the list
+     *
+     * @param command a String[] containing task information stored in the file
+     */
+    private static void remove(String[] command) throws RemoveException {
+        int i = Integer.parseInt(command[1]);
+        if (i > taskList.size()) {
+            throw new RemoveException("That task does not exist");
+        }
+        System.out.println("Understood, I have removed "
+                + taskList.get(i - 1).description);
+        taskList.remove(i - 1);
+        System.out.println("Tasks remaining = " + taskList.size());
+    }
+
+    /**
+     * Adds an event to the list of Tasks
+     *
+     * @param response the string of the user input
+     */
+    private static void Event(String response)
+            throws TimeException {
```

Comment:
Why did you put throws in the next line? Maybe you can be more consistent and put throws at the same place as shown in line 250